### PR TITLE
Inits: Add docstring for main modules

### PR DESCRIPTION
**Pull request by commial**
This PR adds a quick hint on modules behavior.

* `miasm2`: Reverse engineering framework in Python
* `miasm2.analysis`: High-level tools for binary analysis
* `miasm2.arch`: Architectures implementation
* `miasm2.core`: Miasm's core elements
* `miasm2.expression`: Intermediate language implementation
* `miasm2.ir`: Intermediate reprensentation methods
* `miasm2.jitter`: JIT feature
* `miasm2.os_dep`: Operating System dependent methods

May you have better definitions ?

### CODE REVIEW

File: miasm2/jitter/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+"JIT feature"
```
Comment (fmonjalet) [inline miasm2/jitter/__init__.py]: Maybe explain what it is meant for? I think it is more about emulating the real code than about just in time compilation in the end (I am thinking about the python jitter that does not really jit code if I remember well).
Maybe something like : "Code emulation and Just In Time (JIT) recompilation"?

File: miasm2/ir/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+"Intermediate reprensentation methods"
```
Comment (fmonjalet) [inline miasm2/ir/__init__.py]: Typo: reprensentation -> representation
Maybe add that it also contains the symbolic execution methods?

File: miasm2/expression/__init__.py [hunk 3]
```diff
@@ -16,3 +16,4 @@
 # with this program; if not, write to the Free Software Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #
+"Intermediate language implementation"
```
Comment (fmonjalet) [inline miasm2/expression/__init__.py]: To highlight the difference with ir, you could say that this module is about abstract expression manipulation a

**Comment by fmonjalet** [comment]
Hi @commial, that's very nice! I think it can really help newcomers to know what is where, and get an idea of the features.
Maybe the examples from the README could be added to the various module docs, for people who are using help() (or the online doc) to have a quick primer of what's in the module.

**Comment by commial** [comment]
Updated with @fmonjalet approval

**Pull request merged**